src/CodeGenerator/AbstractInjector.php

```diff
@@ -7,6 +7,7 @@
 
 namespace Zend\Di\CodeGenerator;
 
+use function is_string;
```

Comment:
Import functions below classes.

Order for import statements is:

- Classes/namespaces
- Functions
- Constants


And, within each section, alphabetical order by fully qualified name.